src/java.xml.crypto/share/classes/org/jcp/xml/dsig/internal/dom/DOMSignatureMethod.java

```diff
@@ -387,7 +387,7 @@ private static void checkKeySize(XMLCryptoContext context, Key key)
             super(dmElem);
         }
 
-        abstract public PSSParameterSpec getPSSParameterSpec();
+        public abstract PSSParameterSpec getPSSParameterSpec();
```

Comment:
This is 3rd-party (Apache) code. It would be better to change this at Apache. I would leave this one out.

Reply:
@alblue, this is the unresolved comment. Please fix it?